### PR TITLE
LanguageToolBearTest.py: Remove SkipTest warning

### PR DESCRIPTION
**Pull request by rajdeepbharati**
Fixes https://github.com/coala/coala-bears/issues/2796

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by rajdeepbharati** [comment]
Sure @shashank-b

**Comment by rajdeepbharati** [comment]
@shashank-b Could you check the commit message now?

**Comment by rajdeepbharati** [comment]
@shashank-b Should I write 
```
This changes python import statement
from unittest.case import SkipTest --> import unittest.case
It suppresses warning due to __init__ constructor of SkipTest class.
```
in the body of the commit?

**Comment by jayvdb** [comment]
@gitmate-bot rebase

File: tests/natural_language/LanguageToolBearTest.py [hunk 1]
```diff
@@ -14,7 +13,7 @@
     language_check
     guess_language
 except ImportError as err:
-    raise SkipTest(str(err))
+    raise unittest.case.SkipTest(str(err))
```
Comment (jayvdb) [inline tests/natural_language/LanguageToolBearTest.py]: unittest.SkipTest doesnt work?

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2842/commits/e085b3f42a3a7f57925db297ee98d662ea9123e0

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**